accepted/2021/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,380 @@
+# Experimental Features
```

Comment:
I think the experimental features wording is not very encouraging for broader adoption. Our previews engagement is relative low and I think if we talk about committed features as experimental we will experience even lower engagement.

Reply:
Do you have an alternative suggestion? I don't think we should overload the 'preview' term.

In my mind experimental should be fairly well understood, quite a few apps and sites I use have "experimental" features.

Reply:
FWIW, Java calls them [Preview Features])https://openjdk.java.net/jeps/12) but I agree that overloading Preview is a bad idea for us.

Reply:
"Experimental" suggests (to me at least) that the feature may not ever ship. "Preview" infers more commitment that the feature will one day be GA.

If the goal is to increase adoption of these features, I suggest the word implies as strong a commitment as possible. As someone evaluating a feature, the sense that this is actually going to be available in the product in future increases the chance that I try it out, as it reduces my perceived risk that it will be wholesale unavailable beyond that point.

The terms "incubation" or "pre-release" come to mind.

Reply:
We already have a place and shipping vehicle for experiments (experimental features) and it's runtimelab. If we are going to go the extra length and invest in shipping the feature, seek the insightful developer's feedback then we cannot promote it as an experiment otherwise nobody will build anything with it.